src/or.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+// Third Party
+const curry = require("lodash/fp/curry");
+const isFunction = require("lodash/fp/isFunction");
+
+/**
+ * Performs the logical disjunction (or) on the {@code left} or {@code right} disjuncts. If the disjuncts are
+ * functions, they will be evaluated first before comparing.
+ *
+ * @param {*} left - Left conjunct.
+ * @param {*} right - Right conjunct.
+ * @return {Boolean}
+ * @example <caption>Function or Function</caption>
+ * _.or(() => true, () => false);
+ * // => false
+ * @example <caption>Value or Value</caption>
```

Comment:
What if the arguments are `Function`, `Value`?  Should that be added to the documentation?

Reply:
Added documentation and tests to reflect the use of functions.